tests/main/mount-protocol-error/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Check mount bug https://forum.snapcraft.io/t/5682
+
+description: |
+   Regression test for systemd mount race that produces a protocol
+   error. We added a workaround to snapd for the upstream bug
+   https://github.com/systemd/systemd/issues/10872
+
+   For more discussion on the issue see
+   https://forum.snapcraft.io/t/5682
+
+backends: [-autopkgtest]
+
+execute: |
+   for _ in $(seq 50); do
+       snap install test-snapd-tools test-snapd-public
```

Comment:
@mvo5 Is it possible to speed up the test by downloading the snaps and then installing them with --dangerous? This test is taking about 7 minutes to run from a vm in my machine and I guess it will take much more from a borad like pi2.

Reply:
Because of the download cache we implemented recently it should not download the snap multiple times. However I agree we need to target this test much stronger. I.e. we never want this to run on a PI :)

Reply:
I play around with this now, I limited the number of tries to 10 now and also limited the number of systems this will run on (only 18.10, arch and fedora-28 for now). I am checking now how long this takes.